packages/design-system-tokens/src/themes/medicare-component.ts

```diff
@@ -288,7 +292,10 @@ export const components: AnyTokenValues = {
     '-toggle__background-color--hover':           t.color['secondary'],
     '-toggle__color--hover':                      t.color['white'],
     '-toggle__background-color--hover--inverse':  t.color['white'],
-    '-toggle__color--hover--inverse':             t.color['gray-darker'],
+    // I wonder if we can get rid of this variable and just use link-inverse__color--hover
+    // This variable is only used in a medicare style override, and it's weird. Why doesn't
+    // it just behave like a normal inverse link?
+    '-toggle__color--hover--inverse':             t.color['gray-dark'], 
```

Comment:
See the comment as well, but I think we should remove this and the CSS override that it represents. This is what the override behavior looks like, which is weird:

![medicare-drawer-toggle](https://user-images.githubusercontent.com/7595652/218839313-64c10963-1742-4490-892a-b770c121351a.gif)

In my opinion, the colors should just behave like that of a link.

Reply:
@line47, what do you think?

Reply:
Update: We talked about it, and we're going to address it in a separate ticket